crates/telio-proto/src/packet/control/derppoll.rs

```diff
@@ -42,11 +47,16 @@ pub struct DerpPollRequestMsg {
 
 impl DerpPollRequestMsg {
     /// Creates new request message from `peers`.
-    pub fn new(session: Session, peers: &[PublicKey]) -> Self {
+    pub fn new(session: Session, peers: &std::collections::HashSet<PublicKey>) -> Self {
```

Comment:
I would import instead of fully qualifying the name here, especially since we're already importing from `std::collections`

Reply:
Yes. Copy-pasted, forgot to clean up.